usd/src/sdf_parser/Link.cc

```diff
@@ -85,7 +85,7 @@ namespace usd
         return errors;
       }
 
-      if (!pxr::UsdPhysicsRigidBodyAPI::Apply(linkPrim))
+      if (!pxr::UsdPhysicsRigidBodyAPI::Apply(linkPrim.GetParent()))
       {
         errors.push_back(UsdError(sdf::usd::UsdErrorCode::FAILED_PRIM_API_APPLY,
               "Internal error: unable to mark link at path [" + _path
```

Comment:
The error code should be updated to specify the link's parent as the prim that wasn't able to be marked as a rigid body, not the link prim.